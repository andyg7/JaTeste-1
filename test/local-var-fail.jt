

func int main()
{
	int main_var;
	main_var = 10;
	return 0;
}

func void do_something_sick()
{
	int my_var;
	main_var;
}
