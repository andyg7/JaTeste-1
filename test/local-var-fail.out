Scanned
Parsed
Fatal error: exception Exceptions.UndeclaredVariable("main_var")
